src/main/java/com/socialmedia/controller/FileHandlerController.java

```diff
@@ -0,0 +1,28 @@
+package com.socialmedia.controller;
+
+import com.socialmedia.service.AmazonClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequestMapping("/storage/")
```

Comment:
please follow concept of api url convention. Each url should be like '/api/v1/{entity}', so request mapping should be /api/v1/storage